src/lib/DataSourceDefinition/SQLite3.ts

```diff
@@ -1,46 +1,50 @@
 import sqlite3 from "sqlite3";
-import Base from "./Base";
+import Base, { ConfigSchemaType } from "./Base";
 import Util from "../Util";
 
 export default class SQLite3 extends Base {
   db: sqlite3.Database | null;
 
-  static get key() {
+  static get key(): string {
     return "sqlite3";
   }
-  static get label() {
+  static get label(): string {
     return "SQLite3";
   }
-  static get configSchema() {
+  static get configSchema(): ConfigSchemaType {
     return [{ name: "path", label: "Path", type: "string", placeholder: "/path/to/db.sqlite3" }];
   }
 
-  execute(query) {
+  execute(query: string) {
     return this._execute(query);
   }
 
-  async cancel() {
```

Comment:
sqlite_interrupt seems to be synchronized.
https://github.com/mapbox/node-sqlite3/pull/518